radareorg/cutter
# Replace hardcoded system plugin directories with list from Qt.

**Detailed description**

* Replace hard-coded system plugin directories with use of `QStandardPaths` it should return the same values as what was used before but with less platform specific code in Cutter.
* Potentially simplify support of new platforms supported by QT
* Supports gobal plugin directories on Linux. Locations chosen by Qt with any modifications done by distro maintainers.
* Build option to specify additional plugin directories for more pedantic package maintainers that think that plugins are not data.

**Test plan (required)**


* Run on Linux, macOS, Windows and verify that printed paths console output indicates attempts to load from correct directories
    * Linux :heavy_check_mark: 
    * Windows - :heavy_check_mark:  tested by yossizap
    *  macOS - help wanted
* Run AppImage and test that plugins packed inside AppImage are loaded :heavy_check_mark: 



**Closing issues**



Closes #1783
